test/common/alerts_exist.go

```diff
@@ -236,103 +231,100 @@ var expectedRules = []alertsTestRule{
 
 var expectedAWSRules = []alertsTestRule{
 	{
-		File: "redhat-rhmi-operator-connectivity-rule-threescale-redis-example-rhmi.yaml",
+		File: "redhat-rhmi-operator-connectivity-rule-threescale-redis-rhmi.yaml",
```

Comment:
Is the name of the test rule based on the installation name? Because in that case, it would be better to use the `InstallationName` constant

Reply:
Thanks @b1zzu . Sorted now